upgrades/2024_06_12_150527_create_nayra_script_executor.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use ProcessMaker\Models\ScriptExecutor;
+use ProcessMaker\ScriptRunners\Base;
+use ProcessMaker\Upgrades\UpgradeMigration as Upgrade;
+
+class CreateNayraScriptExecutor extends Upgrade
+{
+
+    /**
+     * Run the upgrade migration.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $exists = ScriptExecutor::where('language', Base::NAYRA_LANG)->exists();
+        if (!$exists) {
+            $scriptExecutor = new ScriptExecutor();
+            $scriptExecutor->language = Base::NAYRA_LANG;
+            $scriptExecutor->title = 'Nayra (µService)';
```

Comment:
Is this title is correct? I can see a strange character before "Service" word...